lib/combineMeshes.js

```diff
@@ -1,10 +1,21 @@
 'use strict';
 var Cesium = require('cesium');
+
 var defined = Cesium.defined;
+
 module.exports = combineMeshes;
 
-//Combines meshes in a node. 
-//Should be run before the combinePrimitives stage to merge all primitives for a node into one mesh.
+/**
+ * Combines meshes in a node.
+ * Should be run before the combinePrimitives stage to merge all primitives for a node into one mesh.

```

Comment:
Why not `{@link combinePrimitives}`?

Same comment throughout when it applies.